### ISSUE TITLE
Symbols in the outline pane have the wrong case

### ISSUE THREAD

**Issue by chriscamicas** [chriscamicas/vscode-abl#35]
repro case
```
DEFINE VARIABLE cString       AS CHARACTER NO-UNDO.
DEFINE VARIABLE hSocket       AS HANDLE    NO-UNDO.
DEFINE VARIABLE mBuffer       AS MEMPTR    NO-UNDO.
```
![image](https://user-images.githubusercontent.com/7702519/56039569-d7b29b80-5d34-11e9-87e5-04a5e1abcabf.png)

expected:
symbols in outline should have the same case
```
cString
hSocket
mBuffer
```

**Comment by mscheblein** [comment]
The Changes I made to the Parser for the CodeCompletion also includes a fix for this Bug.

**Comment by chriscamicas** [comment]
fixed in #36
